
CondaError: Run 'conda init' before 'conda activate'

/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py:158: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  print('input size', sys.getsizeof(inputs.storage()))
Traceback (most recent call last):
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/run_hyperparam_testing_random_areas_minibadge.py", line 100, in <module>
    models.RS_hyperparam_tuning_training(train_loader, test_loader, validation_loader, model_type, learning_rate, epochs, log_directory, patience, device, batch_size, sub_batch_size)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py", line 343, in RS_hyperparam_tuning_training
    train.RS_val_model(model, val_loader,  device,  bs, sub_batch_size, writer=writer)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py", line 339, in RS_val_model
    running_test_loss += loss.item()
UnboundLocalError: local variable 'running_test_loss' referenced before assignment
/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py:158: UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  print('input size', sys.getsizeof(inputs.storage()))
Traceback (most recent call last):
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/run_hyperparam_testing_random_areas_minibadge.py", line 100, in <module>
    models.RS_hyperparam_tuning_training(train_loader, test_loader, validation_loader, model_type, learning_rate, epochs, log_directory, patience, device, batch_size, sub_batch_size)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py", line 343, in RS_hyperparam_tuning_training
    train.RS_val_model(model, val_loader,  device,  bs, sub_batch_size, writer=writer)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py", line 339, in RS_val_model
    else:
UnboundLocalError: local variable 'running_test_loss' referenced before assignment
Traceback (most recent call last):
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/run_hyperparam_testing_random_areas_minibadge.py", line 100, in <module>
    models.RS_hyperparam_tuning_training(train_loader, test_loader, validation_loader, model_type, learning_rate, epochs, log_directory, patience, device, batch_size, sub_batch_size)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py", line 343, in RS_hyperparam_tuning_training
    train.RS_train_test_model(model, train_loader, test_loader, lr_rate, def_epochs, log_dir, patience, device, bs, sub_batch_size,writer=writer)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py", line 337, in RS_val_model
    running_test_loss += loss.item()
UnboundLocalError: local variable 'running_test_loss' referenced before assignment
Traceback (most recent call last):
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/run_hyperparam_testing_random_areas_minibadge.py", line 100, in <module>
    models.RS_hyperparam_tuning_training(train_loader, test_loader, validation_loader, model_type, learning_rate, epochs, log_directory, patience, device, batch_size, sub_batch_size)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py", line 344, in RS_hyperparam_tuning_training
    train.RS_val_model(model, val_loader,  device,  bs, sub_batch_size, writer=writer)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py", line 337, in RS_val_model
    running_test_loss += loss.item()
UnboundLocalError: local variable 'running_test_loss' referenced before assignment
Traceback (most recent call last):
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/run_hyperparam_testing_random_areas_minibadge.py", line 100, in <module>
    models.RS_hyperparam_tuning_training(train_loader, test_loader, validation_loader, model_type, learning_rate, epochs, log_directory, patience, device, batch_size, sub_batch_size)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py", line 344, in RS_hyperparam_tuning_training
    train.RS_val_model(model, val_loader,  device,  bs, sub_batch_size, writer=writer)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py", line 337, in RS_val_model
    running_test_loss += loss.item()
UnboundLocalError: local variable 'running_test_loss' referenced before assignment
Traceback (most recent call last):
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/run_hyperparam_testing_random_areas_minibadge.py", line 100, in <module>
    models.RS_hyperparam_tuning_training(train_loader, test_loader, validation_loader, model_type, learning_rate, epochs, log_directory, patience, device, batch_size, sub_batch_size)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/models.py", line 344, in RS_hyperparam_tuning_training
    train.RS_val_model(model, val_loader,  device,  bs, sub_batch_size, writer=writer)
  File "/eejit/home/hausw001/HybGGM/hybGGM_test/src/model/train.py", line 337, in RS_val_model
    running_test_loss += loss.item()
UnboundLocalError: local variable 'running_test_loss' referenced before assignment
